src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -190,6 +196,18 @@ public function __construct(FactoryInterface $factory, Connection $conn)
         $this->factory = $factory;
         $this->valueConverter = $this->factory->get('PHPCR\Util\ValueConverter');
         $this->conn = $conn;
+        $this->namespaces = new \ArrayObject();
+
+        if ($conn->getDatabasePlatform() instanceof PostgreSqlPlatform) {
+            $this->sequenceWorkspaceName = 'phpcr_workspaces_id_seq';
+            $this->sequenceNodeName = 'phpcr_nodes_id_seq';
+            $this->sequenceTypeName = 'phpcr_type_nodes_node_type_id_seq';
+        }
+
+        // @TODO: move to "SqlitePlatform" and rename to "registerExtraFunctions"?
```

Comment:
ah i see. sounds like a good idea to start that PR, but i guess we keep that for a while until we can bump the dbal version to only those having the feature.